admin/tools/docker/deployment/kubernetes/admin/remove-dir.sh

```diff
@@ -1,21 +1,26 @@
 #!/bin/bash
 
-# Remove Qserv data from host nodes
+# Empty given directory on host nodes
 
 # @author Fabrice Jammes SLAC/IN2P3
 
 set -e
 set -x
 
+# Directory to empty
+DATA_DIR="/qserv/log"
```

Comment:
Is this not a mistake? The variable is pointing to a folder where the log files are supposed too be placed.

Reply:
👍 , I'll add a parameter to the script. I use it to remove data or logs files, mostly for running integration tests on openstack ...